src/api/constants/XLSX-CONFIG/index.ts

```diff
@@ -1,78 +1,67 @@
+import { INDEXES, incrementIndexes } from './INDEXES';
 import { POLICY as POLICY_FIELD_IDS } from '../field-ids/insurance/policy';
 import { isMultiplePolicyType } from '../../helpers/policy-type';
-import { Application, XLSXTitleRowIndexes, XLSXRowIndexes } from '../../types';
+import { Application, XLSXRowIndexes } from '../../types';
 
 const {
   TYPE_OF_POLICY: { POLICY_TYPE },
   USING_BROKER,
 } = POLICY_FIELD_IDS;
 
-/**
- * TITLE_INDEXES
- * All XLSX title indexes.
- * @returns {Object}
- */
-export const TITLE_INDEXES = () => ({
-  HEADER: 1,
-  EXPORTER_CONTACT_DETAILS: 10,
-  KEY_INFORMATION: 15,
-  ELIGIBILITY: 21,
-  POLICY: 31,
-  EXPORTER_BUSINESS: 40,
-  BUYER: 58,
-  DECLARATIONS: 66,
-}) as XLSXTitleRowIndexes;
-
-/**
- * INDEXES
- * All XLSX indexes.
- * @returns {Object}
- */
-export const INDEXES = () => ({
-  TITLES: TITLE_INDEXES(),
-  COMPANY_ADDRESS: 34,
-  COMPANY_SIC_CODES: 37,
-  BROKER_ADDRESS: 45,
-  BUYER_ADDRESS: 50,
-  BUYER_CONTACT_DETAILS: 53,
-}) as XLSXRowIndexes;
-
 /**
  * XLSX_ROW_INDEXES
  * Generate row indexes for XLSX.
- * Depending on the submitted application data, the rows can be different:
- * - If the policy type is multiple - the XLSX's "Policy” section has 1 additional row.
- * - If "using a broker" is true - the XLSX's “About your business“ section has 3 additional rows.
- * @returns {Object}
+ * Depending on the submitted application data, the rows can be different.
+ * - If the policy type is multiple, the XLSX has 1 additional row.
+ * - If "using a broker" is true, the XLSX has 3 additional rows.
+ * - If "has different trading address" is true, the XLSX has 1 additional row.
+ * - If "has different trading name" is true, the XLSX has 1 additional row.
+ * - If "has different trading name/address" the ALTERNATIVE_TRADING_ADDRESS row needs to change.
+ * @returns {XLSXRowIndexes}
  */
+
 export const XLSX_ROW_INDEXES = (application: Application): XLSXRowIndexes => {
-  const { policy, broker } = application;
+  const {
+    broker,
+    company: {
+      differentTradingAddress: { fullAddress: hasDifferentTradingAddress },
+      hasDifferentTradingName,
+    },
+    policy,
+  } = application;
 
   const policyType = policy[POLICY_TYPE];
 
-  let isMultiplePolicy = false;
+  let indexes = INDEXES();
 
   if (isMultiplePolicyType(policyType)) {
-    isMultiplePolicy = true;
-  }
-
-  const indexes = INDEXES();
-
-  if (isMultiplePolicy) {
-    indexes.TITLES.EXPORTER_BUSINESS += 1;
     indexes.TITLES.BUYER += 1;
     indexes.TITLES.DECLARATIONS += 1;
 
-    indexes.COMPANY_ADDRESS += 1;
-    indexes.COMPANY_SIC_CODES += 1;
-    indexes.BROKER_ADDRESS += 1;
-    indexes.BUYER_ADDRESS += 1;
     indexes.BUYER_CONTACT_DETAILS += 1;
   }
 
   if (broker[USING_BROKER]) {
+    indexes.TITLES.POLICY += 3;
     indexes.TITLES.BUYER += 3;
     indexes.TITLES.DECLARATIONS += 3;
+    indexes.BUYER_ADDRESS += 3;
+
+    indexes.BROKER_ADDRESS = 48;
```

Comment:
could this be a constant?

Reply:
I'm not sure about this. I think it would make it more confusing, because it's a conditional index, what could the constants be called? There's a few instances of this.

E.g  I think it would be harder to read/follow, compared to having it hard coded here. What do you think?

```ts
indexes.BROKER_ADDRESS = USING_BROKER_BROKER_ADDRESS_INDEX;
indexes.ALTERNATIVE_TRADING_ADDRESS = HAS_DIFFERENT_TRADING_ADDRESS_ALTERNATIVE_ADDRESS_INDEX;

// another condition
indexes.ALTERNATIVE_TRADING_ADDRESS = HAS_DIFFERENT_TRADING_NAME_AND_ADDRESS_ALTERNATIVE_ADDRESS_INDEX;
```

Reply:
Just thought would be easier where would only need to change it in one place.  But this willbe fine as is

Reply:
More than likely to be other instances of this in other sections in the next few tickets, can have another think about it then